### PR TITLE
Sort list of adobe language codes alphabetically.

### PR DESCRIPTION
**Pull request by hlbraddock**
Sort list of adobe language codes alphabetically.
Set each default language by first in order of grouped languages.

### CODE REVIEW

**Comment by hlbraddock** [comment]
@soberstadt

File: lib/adobe/campaign/language.rb [hunk 1]
```diff
@@ -90,104 +88,106 @@ def self.normalize(language_code)
...
         ur_pk
+        ur_in
         uz_uz
         vi_vn
+        zh_cn
+        zh_hk
+        zh_mo
+        zh_sg
+        zh_tw
       ].freeze
```
Comment (soberstadt) [inline lib/adobe/campaign/language.rb:191]: You could have just done
```suggestion
      ].sort.freeze
```

**Review by soberstadt — APPROVED** [review]
This is fine, it could have been easier to sort in ruby when storing it.

File: lib/adobe/campaign/language.rb (hunk 1 above)
Comment (hlbraddock) [reply lib/adobe/campaign/language.rb:191]: That is helpful for future reference. However, in this case, the sort is not entirely alphabetical as this line https://github.com/soberstadt/adobe-campaign/blob/master/lib/adobe/campaign/language.rb#L16 depends on the preferred default language locale be ranked ahead of the other locales for that language, as in https://github.com/soberstadt/adobe-campaign/pull/4/files#diff-0c45cc1aff9fe53797c26a49cb5cd989R94 , which is the primary intent of this PR.

**Pull request merged**